### ISSUE TITLE
Does this support Desktop face tracking?

### ISSUE THREAD

**Issue by ShayBox** [galister/oscavmgr#2]
Does this support Desktop face tracking? It should be possible, but I can't get the MeowFace Android app to work with it.

**Comment by galister** [comment]
We have backends implemented for the following:
- ALVR
- WiVRn
- Project Babble

There's also an ETVR fork somewhere.

For this to work, I believe another backend implementation would be needed.

**Comment by ShayBox** [comment]
What is the backend for? Doesnt this just recieve data from apps and send it to vrchat via osc?

**Comment by galister** [comment]
No, each data source requires a very specific way of retrieving the data.

**Comment by DASPRiD** [comment]
So we'd need a new backend for EyeTrackVR I suppose?

**Comment by galister** [comment]
~~There's eyetrackvr support on the babble backend of this fork here: https://github.com/Coreforge/oscavmgr/tree/etvr_vrcft2~~

Edit: we now support both ETVR and babble on the latest release
